contracts/governance/Staking/WeightedStaking.sol

```diff
@@ -388,7 +388,7 @@ contract WeightedStaking is Checkpoints {
         address account,
         uint256 date,
         uint256 blockNumber
-    ) external view returns (uint96) {
+    ) public view returns (uint96) {
         uint96 priorStake = _getPriorUserStakeByDate(account, date, blockNumber);
         // @dev we need to modify function in order to workaround issue with Vesting.withdrawTokens:
         //		return 1 instead of 0 if message sender is a contract.
```

Comment:
when it used to be called from the Vesting contract & priorStake == 0 it would return 1,
but not in case with the new direct withdraw logic, right?
```
if (priorStake == 0 && isVestingContract(msg.sender))
```

Reply:
Somehow, we need this for governance direct withdrawal as well.
So, i added the workaround by introducing new parameters called `isDirectGovernanceWithdraw`

Reply:
coould be an option but vesting contracts are not upgradeable 
and this fix will break backwards compatibility if it will be called from existing Vesting contracts
pls consider extracting the function code into an internal function and then add an ad-hoc function to be called by Staking but in Vesting contract's context - e.g. `calcForVestingContract`

Reply:
to support backward compatibility, i removed the param. [Commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/892d1be815121b95033677706a1c9cb5677c31e7)